gtfslib/converter.py

```diff
@@ -510,7 +547,7 @@ def import_stop(stop, stoptype, zone_ids, item_ids, station_ids=None):
     logger.info("Normalizing trips...")
     ntrips = 0
     # Process trips and stop times by 1k trips at a time
-    for trip in dao.trips(fltr=(Trip.feed_id == feed_id), prefetch_stop_times=True, prefetch_stops=True, batch_size=800):
+    for trip in dao.trips(fltr=(Trip.feed_id == feed_id), prefetch_stop_times=False, prefetch_stops=False, batch_size=50):
```

Comment:
Does not prefetching stop times have an impact on performance for the normalization step?